### DISCUSSION TITLE
Bisq 2 Release Date

### DISCUSSION THREAD

**Discussion by apemithrandir** [bisq-network/bisq2#1550] [Polls]
I wanted to create this poll to get people's opinions on when Bisq 2 would be released to production.

**Comment by HenrikJannsen** [comment]
Update of dev progress:
Milestone 2.0.0 is our target for release (https://github.com/bisq-network/bisq2/milestone/1)
Many of those issues are in progress and not much missing. The only larger issue which got added recently is a redesign of the trade protocol and trade process UI. I guess that will be a 2-3 weeks task.
I tested quite a bit network stability and it looks pretty stable so far. So I hope there will be no surprise issues coming from that side.

**Comment by Ven32123** [comment]
BISQ2 should be released when it is ready.

**Reply by apemithrandir** [reply]
What does that mean? Will there be a staged release with future feature rollout? What about Bisq Easy?

**Reply by HenrikJannsen** [reply]
Bisq Easy will be in the first release. 

Next roadmap steps are:
- Bisq Easy bugfixing
- Wallet integration
- Multisg protocol
- Submarine swaps
- Porting Multisg to Liquid
- Porting Submarine swaps to Liquid
- Bisq Lightning (combination of Submarine swaps to Liquid and Multisg to Liquid)

**Comment by HenrikJannsen** [comment]
Q1 24 was the winner....
